Doc/library/copyreg.rst

```diff
@@ -29,16 +29,13 @@ Such constructors may be factory functions or class instances.
 
    Declares that *function* should be used as a "reduction" function for objects
    of type *type*.  *function* should return either a string or a tuple
-   containing two or three elements.
+   containing two or three elements. See the :attr:`~pickle.Pickler.dispatch_table`
+   for more details on the interface of *function*.
 
-   The optional *constructor* parameter, if provided, is a callable object which
-   can be used to reconstruct the object when called with the tuple of arguments
-   returned by *function* at pickling time.  A :exc:`TypeError` is raised if the
-   *constructor* is not callable.
+   The *constructor* parameter is a legacy feature and is now ignored, but if
```

Comment:
Just noticed that the param is actually called `constructor_ob`: https://github.com/python/cpython/blob/main/Lib/copyreg.py#L12. Would you mind fixing the docs to use that name?